_maps_2024/30-jack.md

```diff
@@ -0,0 +1,9 @@
+---
+day: 30
+creator: Ji Jin (Jack) Chen
+title: Runeterra
+thumbnail: /assets/thumbnails/2024/30-Jack-FinalMap.png
+source: https://github.com/jijinjc/30DayMappingChallenge2024/blob/main/FinalMap-30.Rmd
+---
+
+This final map was a very interesting map. This map shows one of the best world-buildings in gaming, from League of Legends (do not play the game). In R, many of the aesthetic choices were to color each fictional region to their respective colors, and playing around with the position of texts and images to make it look pretty. Many of the way that images and text are presented here for each region is meant to mimic the original methods used by the people who made this map. Additionally, polygons are simplified because this map is meant to be a little introduction to the region, and thus many of the borders are not exactly the same. 
```

Comment:
```suggestion
This final map was a very interesting map. This map shows one of the best world-buildings in gaming, from League of Legends (do not play the game). In R, many of the aesthetic choices were to color each fictional region to their respective colors, and playing around with the position of texts and images to make it look pretty. Many of the way that images and text are presented here for each region is meant to mimic the original methods used by the people who made this map. Additionally, polygons are simplified because this map is meant to be a little introduction to the region, and thus many of the borders are not exactly the same. 

![Runeterra map](assets/thumbnails/2024/30-Jack-FinalMap.png)
```